pkg/kubelet/kubelet_pods_test.go

```diff
@@ -184,11 +184,23 @@ func TestMakeMounts(t *testing.T) {
 
 func TestNodeHostsFileContent(t *testing.T) {
```

Comment:
This test seems to be only testing `nodeHostsFileContent()` ?
Any test to verify that `hostNetwork=true` hostAlias ?